WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -43,7 +43,7 @@ extension Double {
                 static let units: [Unit] = {
                     var units: [Unit] = []
 
-                    units.append(Unit(abbreviationFormat: NSLocalizedString("%@K", comment: "Label displaying value in thousands. Ex: 66.6K."), accessibilityLabelFormat: NSLocalizedString("%@ thousand", comment: "Accessibility label for value in thousands. Ex: 66.6 thousand.")))
+                    units.append(Unit(abbreviationFormat: NSLocalizedString("%@k", comment: "Label displaying value in thousands. Ex: 66.6k."), accessibilityLabelFormat: NSLocalizedString("%@ thousand", comment: "Accessibility label for value in thousands. Ex: 66.6 thousand.")))
```

Comment:
I went ahead and reverted the change (i.e. make it uppercase again) since I think we all agree on it. If @mattmiklic rebels, I can change it later. 😄

Reply:
No rebellion here, I agree with making this uppercase. :)